api-samples/push/README.md

```diff
@@ -0,0 +1,23 @@
+# self.registration.pushManager.subscribe() userVisibleOnly
+
+This sample demonstrates using the [Push API](https://developer.mozilla.org/en-US/docs/Web/API/Push_API) in chrome with `self.registration.pushManager.subscribe()` and specifically how to use `userVisibleOnly` to silence required notifications when receiving a push message in a service worker based extension.
```

Comment:
I don't think we need this and it allows us to accommodate other Chromium browsers 😄 

```suggestion
This sample demonstrates using the [Push API](https://developer.mozilla.org/en-US/docs/Web/API/Push_API) with `self.registration.pushManager.subscribe()` and specifically how to use `userVisibleOnly` to silence required notifications when receiving a push message in a service worker based extension.
```